pages/osx/security.md

```diff
@@ -13,4 +13,12 @@
 
 - Create a keychain:
 
-`security create-keychain -p {{password}} {{name.keychain}}`
+`security create-keychain -p {{password}} {{keychain.name}}`
+
+- Set a certificate to use with a website or [s]ervice by its [c]ommon name (this will fail if several certificates with the same common name exist):
+
+`security set-identity-preference -s {{URL|hostname|service}} -c {{friendly_certificate_name_in_keychain}} {{path/to/file.keychain}}`
+
+- Add a certificate from file to a [k]eychain (if -k isn't specified, the default keychain is used):
```

Comment:
I don't think, we ever used brackets for indicating this.

```suggestion
- Add a certificate from file to a keychain (if -k isn't specified, the default keychain is used):
```

Reply:
I've observed the [flag_letter] syntax in a number of other pages, so I was following that convention.
e.g.
- /pages/linux/xsel.md:- Use a command's output as input of the **clip[b]oard** (equivalent to `Ctrl + C`):
- /pages/common/chmod.md:- Give **[o]thers** (not in the file owner's group) the same rights as the **[g]roup**:

The output of `grep -r -E '\[[a-zA-Z]\]' . | grep pages/` in the repository root shows this is quite a ubiquitous convention.

Reply:
We may want to revert this commit. You observed correctly.

Reply:
Oh you never added it.. Well that makes it a little easier :smile: